cli/main.go

```diff
@@ -1,7 +1,25 @@
 package main
 
-import "github.com/kosyfrances/rundeck-zabbix/cli/cmd"
+import (
+	"io"
+	"os"
+
+	"github.com/sirupsen/logrus"
+
+	"github.com/kosyfrances/rundeck-zabbix/cli/cmd"
+)
+
+const logFile = "/tmp/rundeck-zabbix.log"
```

Comment:
How about running in WIN systems?
What do you think of having the log path to be in the same folder as the built binary?

Reply:
I've not thought of WIN systems yet. Maybe that can be an added feature later on? At the moment the focus is on Linux. About the same folder as the built binary, not sure how to track that either.

Reply:
This can be configurable with a  specified default but this is a decent place to start and subsequently iterate